### PR TITLE
Cnv api style test

### PR DESCRIPTION
**Pull request by aburdenthehand**
Creating dummy origin PR to get some second opinions on exceptions

### CODE REVIEW

File: modules/virt-checking-storage-class.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ Cinder, the default storage class, does not support VM import. See link:https://
 
 You can check the default storage class in the {product-title} console. If the default storage class is not NFS, you can change the default storage class so that it is no longer the default and change the NFS storage class so that it is the default.
 
-If more than one default storage class is defined, the VirtualMachineImport CR uses the default storage class that is first in alphabetical order.
+If more than one default storage class is defined, the virtual machine import CR uses the default storage class that is first in alphabetical order.
```
Comment (aburdenthehand) [inline modules/virt-checking-storage-class.adoc]: Not sure about this one

File: modules/virt-confirming-policy-updates-on-nodes.adoc [hunk 2]
```diff
@@ -3,39 +3,39 @@
 // * virt/node_network/virt-configuring-node-network-policy
 
 [id="virt-confirming-policy-updates-on-nodes_{context}"]
-= Confirming Policy updates on nodes
+= Confirming policy updates on nodes
```
Comment (aburdenthehand) [inline modules/virt-confirming-policy-updates-on-nodes.adoc]: This might need to be reworded now to be more specific

File: modules/virt-troubleshooting-incorrect-policy-config.adoc [hunk 3]
```diff
@@ -3,19 +3,19 @@
 // * virt/node_network/virt-troubleshooting-node-network.adoc
 
 [id="virt-troubleshooting-incorrect-policy-config_{context}"]
-= Troubleshooting an incorrect NodeNetworkConfigurationPolicy configuration
+= Troubleshooting an incorrect node network configuration policy configuration
 
 You can apply changes to the node network configuration across your entire cluster by applying a
 NodeNetworkConfigurationPolicy. If you apply an incorrect configuration, you can use the
```
Comment (aburdenthehand) [inline modules/virt-troubleshooting-incorrect-policy-config.adoc]: Needs to be manually changed

File: modules/virt-setting-policy-attributes.adoc [hunk 4]
```diff
@@ -25,4 +25,4 @@ spec:
         policy: require <2>
 ----
 <1> Name of the CPU feature for the VM or VMI.
-<2> Policy attribute for the VM or VMI.
+<2> policy attribute for the VM or VMI.
```
Comment (aburdenthehand) [inline modules/virt-setting-policy-attributes.adoc]: Naughty script.

File: modules/virt-about-conditions-and-events.adoc [hunk 5]
```diff
@@ -5,7 +5,7 @@
 [id="virt-about-conditions-and-events.adoc_{context}"]
 = About conditions and events
 
-Diagnose DataVolume issues by examining the output of the `Conditions` and `Events` sections
+Diagnose data volume issues by examining the output of the `Conditions` and `Events` sections
```
Comment (ousleyp) [inline modules/virt-about-conditions-and-events.adoc]: I do find "data volume" to seem overly generic. My first thought is to wonder if we should mandate always writing it out like this:

```
`DataVolume` object(s)
```

But that would seems too wordy, especially in places where we have pluralized "DataVolume" in the past.  

And in this particular example, "Diagnose `DataVolume` object issues" sounds awkward. "Issues with `DataVolume` objects" also sounds awkward.

My second thought is... can we get away with just writing it "datavolume(s)" without the space, when we are not referring to a specific object?

File: modules/virt-about-upgrading-virt.adoc [hunk 6]
```diff
@@ -28,14 +28,14 @@ virtual machine.
...
-instance. The `virt-launcher` Pod runs an instance of `libvirt`, which is
+Each virtual machine has a `virt-launcher` pod that runs the virtual machine
+instance. The `virt-launcher` pod runs an instance of `libvirt`, which is
 used to manage the virtual machine process.
 ====
 
 * Upgrading does not interrupt network connections.
 
-* DataVolumes and their associated PersistentVolumeClaims are preserved during
+* data volumes and their associated persistent volume claims are preserved during
```
Comment (ousleyp) [inline modules/virt-about-upgrading-virt.adoc]: If there's a bullet point `* ` preceding the term, we should capitalize the first letter.

File: modules/virt-checking-storage-class.adoc (hunk 1 above)
Comment (ousleyp) [reply modules/virt-checking-storage-class.adoc]: I think if you're saying "custom resource," it's like saying "object," so it should be `VirtualMachineImport`

File: modules/virt-confirming-policy-updates-on-nodes.adoc (hunk 2 above)
Comment (ousleyp) [reply modules/virt-confirming-policy-updates-on-nodes.adoc]: +1

File: modules/virt-creating-new-vm-from-cloned-pvc-using-datavolumetemplate.adoc [hunk 7]
```diff
@@ -3,19 +3,19 @@
...
-a DataVolume, which is then automatically used for the creation of the virtual
+a data volume, which is then automatically used for the creation of the virtual
 machine.
 
 [NOTE]
 ====
-When a DataVolume is created as part of the DataVolumeTemplate of a virtual
-machine, the lifecycle of the DataVolume is then dependent on the virtual
-machine. If the virtual machine is deleted, the DataVolume and associated
+When a data volume is created as part of the data volumeTemplate of a virtual
```
Comment (ousleyp) [inline modules/virt-creating-new-vm-from-cloned-pvc-using-datavolumetemplate.adoc]: Alas, `DataVolumeTemplate` is another one to explicitly look for

File: modules/virt-deleting-deployment-custom-resource.adoc [hunk 8]
```diff
@@ -5,7 +5,7 @@
 //This file contains UI elements and/or package names that need to be updated.
 
 [id="virt-deleting-deployment-custom-resource_{context}"]
-= Deleting the {VirtProductName} Operator Deployment custom resource
+= Deleting the {VirtProductName} Operator deployment custom resource
```
Comment (ousleyp) [inline modules/virt-deleting-deployment-custom-resource.adoc]: In this case, "Deployment" is capitalized because that's how it appeared in the UI

File: modules/virt-deleting-virt-cli.adoc [hunk 9]
```diff
@@ -16,7 +16,7 @@ You can delete {VirtProductName} by using the CLI.
 
 [NOTE]
 ====
-When you delete the subscription of the {VirtProductName} operator in the OLM by using the CLI, the ClusterServiceVersion (CSV) object is not deleted from the cluster. To completely uninstall {VirtProductName}, you must explicitly delete the CSV.
+When you delete the subscription of the {VirtProductName} operator in the OLM by using the CLI, the cluster service version (CSV) object is not deleted from the cluster. To completely uninstall {VirtProductName}, you must explicitly delete the CSV.
```
Comment (ousleyp) [inline modules/virt-deleting-virt-cli.adoc]: This says "object," so I think it would be `ClusterServiceVersion`. Another edge case to look for: parentheticals interrupting the phrase

File: modules/virt-importing-vm-cli.adoc [hunk 10]
```diff
@@ -5,7 +5,7 @@
 [id="virt-importing-vm-cli_{context}"]
 = Importing a virtual machine with the CLI
 
-You can import a virtual machine with the CLI by creating the Secret and VirtualMachineImport Custom Resources (CRs). The Secret CR stores the RHV Manager credentials and CA certificate. The VirtualMachineImport CR defines the parameters of the VM import process.
+You can import a virtual machine with the CLI by creating the secret and virtual machine import Custom Resources (CRs). The secret CR stores the RHV Manager credentials and CA certificate. The virtual machine import CR defines the parameters of the VM import process.
```
Comment (ousleyp) [inline modules/virt-importing-vm-cli.adoc]: Again, since these are described as custom resources, they should get the backticks/pascalcase ("custom resources" should not be capitalized, though)

File: modules/virt-importing-vm-cli.adoc [hunk 11]
```diff
@@ -76,7 +76,7 @@ EOF
 ----
 <1> Specify the RHV logical network and vNIC profile.
 <2> Specify the {VirtProductName} network.
-<3> If `storageMappings` are specified in both the ResourceMapping and the VirtualMachineImport CRs, the VirtualMachineImport CR takes precedence.
+<3> If `storageMappings` are specified in both the ResourceMapping and the virtual machine import CRs, the virtual machine import CR takes precedence.
```
Comment (ousleyp) [inline modules/virt-importing-vm-cli.adoc]: Same as above -- anywhere we see CR or custom resource or Custom Resource should get the backticks/pascalcase

File: modules/virt-setting-policy-attributes.adoc (hunk 4 above)
Comment (ousleyp) [reply modules/virt-setting-policy-attributes.adoc]: Good catch. :)

**Review by ousleyp — COMMENTED** [review]
Some thoughts and observations -- thanks for taking on this effort, and let me know if I can help manually channge anything!

File: modules/virt-deleting-deployment-custom-resource.adoc (hunk 8 above)
Comment (aburdenthehand) [reply modules/virt-deleting-deployment-custom-resource.adoc]: Thanks for the catch :)

File: modules/virt-about-upgrading-virt.adoc (hunk 6 above)
Comment (aburdenthehand) [reply modules/virt-about-upgrading-virt.adoc]: :clinking_glasses:
